oemof_b3/tools/testing_pipeline.py

```diff
@@ -163,83 +181,169 @@ def get_abs_path_list(output_rule_list):
 
     Outputs
     -------
-    absolute_path_list : str
-        Absolute file path
+    absolute_path_list : list
+         Absolute file path in list
 
     """
-    # Loop over each rule which is tested in the snakemake pipeline
-    for sublist in output_rule_list:
-        # Get absolute path
-        absolute_path_list = [os.path.join(os.getcwd(), entry) for entry in sublist]
+    # Get absolute path of rule
+    absolute_path_list = [os.path.abspath(entry) for entry in output_rule_list]
 
     return absolute_path_list
 
 
+def file_name_extension(raw_file_path):
+    """
+    This function rearranges the current absolute file path
+    with the new extension suffix '_original'.
+
+    Inputs
+    -------
+    raw_file_path : str
+        Absolute path of rule
+
+    Outputs
+    -------
+    renamed_path : str
```

Comment:
Could you rename the output to renamed_file_path for consistency reasons please?